<launch>
  <node pkg="urg_node" type="urg_node" respawn="false" name="urg_node" output="screen">
    <param name="ip_address " type="string" value="192.168.0.10" />
    <param name="frame_id " type="string" value="laserScan" />
    <param name="publish_intensity  " type="bool" value="false" />
    <param name="angle_min" type="double" value="-&pi" />
    <param name="angle_max" type="double" value="&pi" />
  </node>
</launch>
